### DISCUSSION TITLE
Keyboard Shortcut?

### DISCUSSION THREAD

**Discussion by Qhilm** [scambier/obsidian-omnisearch#162] [Q&A] [answered]
I must be blind, I searched for it, I cannot find how to invoke omnisearch with the keyboard. Can someone help me out?

**Comment by scambier** [comment, marked as answer]
Hotkeys are managed directly within Obsidian:
![image](https://user-images.githubusercontent.com/3216752/209117828-68fa7e4c-11f1-4e26-8633-26339c432691.png)
